src/batchUploader.ts

```diff
@@ -233,6 +287,8 @@ export class BatchUploader {
                 let blob = new Blob([fetchPayload.body], {
                     type: 'text/plain;charset=UTF-8',
                 });
+                // TODO: Can we flag anytihng in the vault to say it's
```

Comment:
i think this would be helpful

Reply:
We should do this as part of a future revision to prevent scope creep in this PR.